Analysis/analyze.py

```diff
@@ -123,7 +123,49 @@ def look_for_within(container_image, sub_image, start_x, start_y, x_range, y_ran
 
             if (res > 0):
                 return (i, j)
-	return None
+    return None
+
+#Convert one rgb pixel to lab
+def Pxl_rgb2lab(pixel): #pixel will be a 4-D array RGBa (a ignored)
+    #Convert to XYZ
+    ref_Whites = numpy.array([95.047, 100.0, 108.883, 1])
+    for i in range(3):
+        val = pixel[i] / 255
+        if val > 0.04045:
```

Comment:
Let's make this a constant. Better yet, we should make this conversion library its own class (separate file), instantiate objects of it using our pixel/matrices, and call into it.